Makefile

```diff
@@ -30,12 +30,19 @@ install: install-scripts install-conf install-exclude install-systemd
 # target: install-scripts - Install executables.
 install-scripts:
 	install -d $(DEST_SCRIPTS)
-	install -m 744 $(SRCS_SCRIPTS) $(DEST_SCRIPTS)
+	install -m 0744 $(SRCS_SCRIPTS) $(DEST_SCRIPTS)
+
+etc/restic/b2_env.sh:
+	install -m 0600 etc/restic/b2_env.sh.template etc/restic/b2_env.sh
+
+etc/restic/b2_pw.txt:
+	install -m 0600 etc/restic/b2_pw.txt.template etc/restic/b2_pw.txt
 
 # target: install-conf - Install restic configuration files.
-install-conf:
-	install -d $(DEST_CONF) -m 700
-	install $(SRCS_CONF) $(DEST_CONF)
+# will create these files locally only if they don't already exist
+install-conf: | etc/restic/b2_env.sh etc/restic/b2_pw.txt
```

Comment:
the pipe delimiter indicates that the succeeding files are [order-only prerequisites](https://stackoverflow.com/a/21754930/684619), meaning they will only be made if they don't already exist